lib/pages/child_menu/customizable_column.dart

```diff
@@ -1,68 +1,76 @@
+import 'dart:async';
+import 'package:fake_cloud_firestore/fake_cloud_firestore.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
 import 'package:firebase_auth/firebase_auth.dart';
+import 'package:firebase_auth_mocks/firebase_auth_mocks.dart';
+import 'package:firebase_storage/firebase_storage.dart';
+import 'package:firebase_storage_mocks/firebase_storage_mocks.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
+import 'package:seg_coursework_app/data/choice_boards_data.dart';
+import 'package:seg_coursework_app/models/category.dart';
+import 'package:seg_coursework_app/models/clickable_image.dart';
+import 'package:seg_coursework_app/pages/admin/admin_choice_boards.dart';
+import 'package:seg_coursework_app/services/auth.dart';
+import '../../helpers/firebase_functions.dart';
+import '../../models/categories.dart';
+import 'package:flutter/services.dart';
 import 'package:seg_coursework_app/pages/admin/admin_choice_boards.dart';
 import 'package:seg_coursework_app/services/auth.dart';
 import 'package:shared_preferences/shared_preferences.dart';
 import 'customizable_row.dart';
 
 // The child menu is formed essentially by creating a column of rows,
 // with rowConfigs outlining the category's title and images
-// (1st image is category image and those after are the smaller previews
-//
 
 class CustomizableColumn extends StatefulWidget {
   final bool mock;
+  static int customizableColumnRequestCounter = 0; //used for testing
+  List<List<ClickableImage>> testList; //used for testing
+  CustomizableColumn({this.mock = false, testList})
```

Comment:
why is the field here called testList. Is it used for testing only? Otherwise please rename